code/_globalvars/lists/flavor_misc.dm

```diff
@@ -173,8 +173,12 @@ GLOBAL_LIST_INIT(jumpsuitlist, list(PREF_SUIT, PREF_SKIRT))
 #define UPLINK_PDA "PDA"
 #define UPLINK_RADIO "Radio"
 #define UPLINK_PEN "Pen" //like a real spy!
-#define UPLINK_IMPLANT  "Implant"
-GLOBAL_LIST_INIT(uplink_spawn_loc_list, list(UPLINK_PDA, UPLINK_RADIO, UPLINK_PEN, UPLINK_IMPLANT))
+#define UPLINK_IMPLANT "Implant"
+#define UPLINK_IMPLANT_WITH_PRICE "[UPLINK_IMPLANT] (-[UPLINK_IMPLANT_TELECRYSTAL_COST] TC)"
```

Comment:
This is a horrible way of doing it, now if we ever change the price we have to factor in that it'll fuck with people's data.

Keep the keys the same, make the input ITSELF display something different.

Reply:
No, the whole idea is that it will NOT mess with people's data. Only "Implant" is saved.